avocado_vt/test.py

```diff
@@ -253,6 +253,15 @@ def setUp(self):
             details = sys.exc_info()[1]
             self.__status = details
         finally:
+            # Clean libvirtd debug logs if the test is not fail or error
+            if(self.params.get("vm_type") == 'libvirt' and
+               self.params.get("enable_libvirtd_debug_log", True)):
```

Comment:
can we change the value to `yes`, `no` and lets check `self.params.get("enable_libvirtd_debug_log", "yes") == "yes"`

Reply:
sure, Agreed.